seed/static/seed/js/controllers/menu_controller.js

```diff
@@ -147,6 +150,39 @@ angular.module('BE.seed.controller.menu', [])
         });
       };
 
+      /**
+       * open_sample_data_modal: opens the auto-populate sample data modal
+       */
+      $scope.open_sample_data_modal = function () {
+        $uibModal.open({
+          templateUrl: urls.static_url + 'seed/partials/sample_data_modal.html',
+          controller: 'sample_data_modal_controller',
+          size: 'md',
+          resolve: {
+            organization: () => {
+              return $scope.menu.user.organization;
+            },
+            cycle: ['organization_service', function (organization_service) {
+              return organization_service.get_organization($scope.menu.user.organization.org_id)
+                .then(response => {
+                  let lastCycleId = inventory_service.get_last_cycle();
+                  let lastCycle;
+                  if (typeof lastCycleId === 'number') {
+                    lastCycle = response.organization.cycles.find(cycle => cycle.cycle_id === lastCycleId)
+                  }
+                  if ((lastCycleId === undefined || !lastCycle) && response.organization.cycles.length) {
+                    lastCycle = response.organization.cycles[0];
+                  }
+                  return lastCycle;
+                });
+            }],
```

Comment:
This assumes the existence of a cycle. The homepage's "Auto-Populate Sample Data" button doesn't work (the modal won't show) if we've deleted all of our cycles. This is more of an edge case, but we do display the warning message of "To continue you must first remove pre-existing data", which doesn't specify what all should be deleted. I doubt a new user would think to delete a cycle.

Anyway, knowing it's an edge case, I'll defer to others on whether we want to make change here, but here are a few considerations/options:
- If we want to allow the auto-populate feature even without a Cycle, the sample data modal would need to be updated to not need a cycle. And the insert_sample_data endpoint would would need to be updated to create a cycle.
- Alternatively, we could check for a cycle in this function before opening the modal and, if none exists, prompt the user to create a cycle via Notification.